### PR TITLE
tested the code

### PR DESCRIPTION
**Pull request by Ade179**

### CODE REVIEW

File: src/__mock__/addandremove.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+describe('should addandremove a task', () => {
+  test('checking for task removal', () => {
+    const arr = [2, 3, 5];
+    expect(arr.splice(1, 2)).toHaveLength(2);
+  });
+  test('checking for task addition', () => {
+    const arr = [3];
+    expect(arr.length).toBe(1);
+  });
+});
```
Comment (DammyShittu) [inline src/__mock__/addandremove.test.js:10]: - [ ] Based on the requirements you are required to write tests for the **specific** methods responsible for adding and removing a task, but that is not the case here. You need to implement this addition and removal using your custom methods, not inbuilt JS methods.

- [ ] Kindly mock the local storage or DOM and ensure the add and remove methods add and remove tasks from them.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Ade179 and @Osoro254Alex,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__mock__/addandremove.test.js (hunk 1 above)
Comment (Ade179) [reply src/__mock__/addandremove.test.js:10]: thanks, but please, could you kindly explain further and maybe suggest how to approach these changes @DammyShittu

**Review by topeogunleye — APPROVED** [review]
### Hi @Ade179, @Osoro254Alex, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention. 
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**